docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -221,7 +228,6 @@ In this section we will do some manual changes that will allow you to check AWS
 vim src/test/system_tests/ceph_s3_tests/test_ceph_s3_config_aws.conf
 ```
 * host = s3.amazonaws.com
-* bucket prefix = choose_name (for example: `bucket prefix = foo-bucket` you will need to manually delete it from AWS, and its name will be `foo-bucket1`, it adds suffix of 1).
```

Comment:
We can still use this comment with a small change: add **in case** you will need...